runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -170,7 +171,7 @@ public void addNode(Layout currentLayout,
      * @throws OutrankedException         if consensus is outranked.
      */
     public boolean mergeSegments(Layout currentLayout)
-            throws QuorumUnreachableException, OutrankedException, ExecutionException {
+            throws QuorumUnreachableException, OutrankedException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.runtime.exceptions.QuorumUnreachableException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)